server/repositories/cmsQueries/categoryCollectionsQuery.js

```diff
@@ -1,21 +1,30 @@
 const { DrupalJsonApiParams: Query } = require('drupal-jsonapi-params');
 const { getSmallTile, getPagination } = require('../../utils/jsonApi');
 
-class AllSeriesQuery {
+class CategoryCollectionsQuery {
```

Comment:
Naming Q? here - does this refer to a component on the frontend? might be better to name it that rather than a drupal specific terminology